### PR TITLE
PEM - 9.1 Release branch

### PR DESCRIPTION
**Pull request by nidhibhammar**
Added file for 9.1 release notes as per PEM-4730

### CODE REVIEW

File: product_docs/docs/pem/9/pem_rel_notes/910_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+| Type        | Description                                                                                                                                                  | ID                 |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------ |
+| Enhancement | Support for EDB Advanced Server 15 and EDB Postgres Distributed 5.                                                                                           | PEM-4640           |
+| Enhancement | Added the ability to copy probes and alerts to all servers in a group without having to select them individually.                                            | PEM-4613, PEM-4681 |
+| Bug Fix     | Fixed an issue whereby alert notifications were sent for Low or Medium alerts when the user had selected not to receive them. [Support Ticket #87664]        | PEM-4650           |
+| Bug Fix     | Fixed an issue whereby webhooks would not be called if there was an inactive endpoint in the spool. [Support Ticket #87365]                                  | PEM-4664           |
```
Comment (sonotley) [inline product_docs/docs/pem/9/pem_rel_notes/910_rel_notes.mdx:12]:
```suggestion
| Bug Fix     | Fixed an issue whereby webhooks would not be called if there was an inactive endpoint in the spool. [Support Ticket #87365]                                  | PEM-4664           |
| Other     | The PGD Worker Errors probe and associated alert template and dashboard chart have been removed as they were found to produce incorrect data. They will be replaced with a new probe in a future release. ||
```
Adding a line for a last-minute change

File: product_docs/docs/pem/9/troubleshooting.mdx [hunk 2]
```diff
@@ -38,6 +39,15 @@ dnf remove python3-mod_wsgi
...
 
+### RHEL 7 ppc64le
+
+After installing the PEM server on RHEL 7 ppc64le, you may see the following cipher error in the worker log file:
+
+```text
+WARNING: ConnectToPEM: unable to connect to PEM database: could not create SSL context: library has no ciphers
+```
+
+To resolve this error, you must upgrade your PostgreSQL to version 12.x, where x is the latest available version. Then, restart the PEM agent.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/troubleshooting.mdx:50]:
```suggestion
To resolve this error, if you intend to stay on version PostgreSQL version 12, upgrade to the latest available version. Then, restart the PEM agent.
```

File: product_docs/docs/pem/9/troubleshooting_server.mdx [hunk 3]
```diff
@@ -12,6 +12,8 @@ redirects:
 
 ##  Server installation errors
 
+### RHEL 8
+
 While installing the PEM server on RHEL 8, you might see this error:
```
[the remark this answers is not in the document]
Comment (arup-roy2023) [reply product_docs/docs/pem/9/troubleshooting_server.mdx]: Done

File: product_docs/docs/pem/9/troubleshooting_server.mdx [hunk 4]
```diff
@@ -37,6 +39,15 @@ dnf remove python3-mod_wsgi
 
 Then try installing the PEM server again.
```
[the remark this answers is not in the document]
Comment (arup-roy2023) [reply product_docs/docs/pem/9/troubleshooting_server.mdx]: Applied changes

File: product_docs/docs/pem/9/troubleshooting_server.mdx [hunk 5]
```diff
@@ -37,6 +39,15 @@ dnf remove python3-mod_wsgi
...
 
+### RHEL 7 ppc64le
+
+After installing the PEM server on RHEL 7 ppc64le, you may see the following cipher error in the worker log file:
+
+```text
+WARNING: ConnectToPEM: unable to connect to PEM database: could not create SSL context: library has no ciphers
+```
+
+To resolve the error, you must upgrade your PostgreSQL to version 12.4 or later and then restart the PEM agent.
```
[the remark this answers is not in the document]
Comment (arup-roy2023) [reply product_docs/docs/pem/9/troubleshooting_server.mdx]: Done

**Review by arup-roy2023 — APPROVED** [review]

**Pull request merged**